packages/react-wildcat/src/server.js

```diff
@@ -29,6 +29,9 @@ let server;
 
 function start() {
     const wildcatConfig = require("./utils/getWildcatConfig")(cwd);
+    if (process.env.DEBUG === "true") {
```

Comment:
Consider using the [debug](https://github.com/visionmedia/debug) module for this to avoid clobbering the `DEBUG` env var.